packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/Table/DeleteButton/index.js

```diff
@@ -28,6 +32,12 @@ const DeleteButton = ({ tokenName, onClickDelete }) => {
         noBorder
         icon={<Trash />}
       />
+      {showConfirmDialog && (
```

Comment:
Why are you conditionally rendering this when there a props that handle the visibility of it?

Reply:
Also: could you please add a test? :)